docs/upgrading.md

```diff
@@ -53,6 +55,71 @@
     - [Removing unnecessary metadata using the resourceprocessor](#removing-unnecessary-metadata-using-the-resourceprocessor)
     - [Moving record-level attributes used for metadata to the resource level](#moving-record-level-attributes-used-for-metadata-to-the-resource-level)
 
+
+## Upgrading to v0.103.0-sumo-0
+
+### `sumologic` configuration: modified the `configuration files` merge behaviour
+
+Modified the configuration merge behaviour to perform overwrite instead of update for collector_fields key.
```

Comment:
```suggestion
Modified the configuration merge behaviour to perform overwrite instead of update for `collector_fields` key of [Sumo Logic Extension](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/extension/sumologicextension#configuration).
```